awx/main/migrations/0164_remove_inventorysource_update_on_project_update.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 3.2.13 on 2022-06-21 21:29
+
+from django.db import migrations
+import logging
+
+logger = logging.getLogger("awx")
+
+
+def forwards(apps, schema_editor):
+    InventorySource = apps.get_model('main', 'InventorySource')
+    sources = InventorySource.objects.filter(update_on_project_update=True)
+    for src in sources:
+        if src.update_on_launch == False:
+            src.update_on_launch = True
+            src.save(update_fields=['update_on_launch'])
```

Comment:
it seems a little in-congruent to not have a log here (like for the project change). Even if it's a lower level, like `logger.info`, that would be good to add.